.travis.yml

```diff
@@ -13,6 +13,13 @@ script:
   # Only run the targets if we are on the same OS.
   - if [ "$RUN" == "1" ]; then cargo test --verbose --target "$TARGET"; fi
 
+env:
+  global:
+    # Per https://docs.travis-ci.com/user/reference/overview/ -
+    # Travis CI servers make 2 cores available to processes via
+    # virtualization.
+    - EXPECTED_TRAVIS_CPU_COUNT=2
```

Comment:
If `NUM_CPUS_TEST_GET=2` and `NUM_CPUS_TEST_GET_PHYSICAL=16` were set, then there wouldn't need to be a specific test for Travis in the source code at all. Sorry if it wasn't clear that I meant using those existing env vars...

Reply:
Ahh, gotcha! Yeah that's way cleaner. Changed in b800eb531a68fd48508009ccbf08674564ae7f6a